src/hotspot/share/compiler/oopMapStubGenerator.hpp

```diff
@@ -51,3 +54,7 @@ class OopMapStubGenerator {
 };
 
 #endif
+
+#else
+#error This code is not used
+#endif
```

Comment:
Can you check Windows, I suspect it might have to be excluded.

Reply:
Um, Windows builds fail here:
 https://github.com/shipilev/loom/runs/3716995428?check_suite_focus=true

Which implies `oopMapStubGenerator_x86.cpp` is still compiled? What. Let me see...

Reply:
See #68.